pnpm-workspaces/packages/one/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
+load("@aspect_rules_js//npm:defs.bzl", "npm_package")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+
+npm_link_all_packages(name = "node_modules")
+
+ts_project(
+    name = "one_ts",
+    srcs = ["src/main.ts"],
+    tsconfig = "//:tsconfig",
+    deps = [
+        "//:node_modules/@types/node",
```

Comment:
Are any node APIs being used that require this?

Reply:
Yes, the use of `console` requires it:

```
packages/one/src/main.ts(2,3): error TS2584: Cannot find name 'console'. Do you need to change your target library? Try changing the 'lib' compiler option to include 'dom'.
```

Reply:
👍